src/element/owned.rs

```diff
@@ -598,6 +598,18 @@ impl From<Symbol> for Value {
     }
 }
 
+impl From<&[u8]> for Value {
+    fn from(value: &[u8]) -> Self {
+        Value::Blob(value.into())
+    }
+}
+
+impl From<Vec<u8>> for Value {
+    fn from(value: Vec<u8>) -> Self {
+        Value::Blob(value)
+    }
+}
```

Comment:
🗺️ While writing the doc tests, I realized we didn't have an `Into<Value>` impl for the `blob`-ish Rust types.